.github/workflows/build.yml

```diff
@@ -1,6 +1,6 @@
 on:
   push:
-    branches: [master, parachain]
+    branches: [master, parachain, ci/fix-1.62.0-not-found]
```

Comment:
Indeed, done! thanks. It would be great to have a way of manually executing the CI flows that only run for `parachain` on other branches, to avoid the need to change code. Something like `\bot run full-ci`.